lib/Service/RoomFormatter.php

```diff
@@ -162,6 +162,7 @@ public function formatRoomV4(
 			return array_merge($roomData, [
 				'name' => $room->getName(),
 				'displayName' => $room->getDisplayName($isListingBreakoutRooms || $isSIPBridgeRequest || $this->userId === null ? '' : $this->userId, $isListingBreakoutRooms || $isSIPBridgeRequest),
+				'description' => $room->getDescription(),
```

Comment:
```suggestion
				'description' => $room->getListable() !== Room::LISTABLE_NONE ? $room->getDescription() : '',
```
But I will take care of it when adding an integration test